packages/block-library/src/gallery/style.scss

```diff
@@ -57,6 +57,12 @@
 			font-size: $default-font-size;
 			background: linear-gradient(0deg, rgba($color: $black, $alpha: 0.7) 0, rgba($color: $black, $alpha: 0.3) 70%, transparent);
 
+			// White background variation
+			.is-style-white-caption-background & {
+				color: inherit;
```

Comment:
I [think](https://github.com/WordPress/gutenberg/issues/24896#issuecomment-683660816) we could add user-selected colors for the caption and links within. In that case, would it make sense that this color is opinionated (black-ish) if the user hasn't selected any color?

Reply:
I do like the idea of the user providing the colors and having some defaults. 

The gradient has actually been slightly headachy because it doesn't work with multiple lines. Not that we can change it easily, but I do like the idea of Global Styles expanding the palette here.